MantidQt/CustomInterfaces/src/SampleTransmission.cpp

```diff
@@ -184,8 +184,8 @@ void SampleTransmission::algorithmComplete(bool error) {
   m_uiForm.twResults->addTopLevelItem(transmissionItem);
   transmissionItem->setExpanded(true);
 
-  std::vector<double> transmissionData = ws->dataY(0);
-  Statistics stats = getStatistics(transmissionData);
+  auto &transmissionData = ws->y(0);
+  Statistics stats = getStatistics(transmissionData.rawData());
```

Comment:
`getStatistics(ws->y(0).rawData())`, the transmissionData variable is not used below